internal/app/initapp/initapp.go

```diff
@@ -0,0 +1,46 @@
+package initapp
```

Comment:
```suggestion
package app
```
Пакет app а метод Init , так логичней. Если захочется добавить другой метод не придется переименовывать файл